cypress.env.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We don't typically commit developer env files, right? I would instead add instructions in the readme for the needed env setup, possibly including a `.example` config file.  Also, `dhis2_datatest_prefix` is not in use.

Reply:
Removed and added to list in `gitignore`